packages/universal-components/src/RangeDatePicker/RangeDatePicker.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { Text } from '../Text';
+import { Modal } from '../Modal';
+import { Touchable } from '../Touchable';
+import { StyleSheet } from '../PlatformStyleSheet';
+import type { Props } from './RangeDatePickerTypes';
+import RangeDatePickerContent from './RangeDatePickerContent';
+
+type State = {
+  date: Date,
+};
+
+const parsePropsToState = ({ date }: Props) => {
```

Comment:
This naming is a little confusing for me, from that name, I will be expected that will parse all props to state. So what about `parseDatePropsToState` or something more descriptive? But it is just Idea

Reply:
Sounds good, done